backend/sozisel/test/sozisel_web/schema/events/quizzes/quiz_mutations_test.exs

```diff
@@ -0,0 +1,256 @@
+defmodule SoziselWeb.Schema.QuizMutationsTest do
```

Comment:
IMPORTANT

In elixir module name should represent its location inside project. So if you added `events` directory then it should be represented in module name as `.Events.`

Reply:
Okay